### PR TITLE
Create ashish_test

### PR DESCRIPTION
**Pull request by Ashish503-web**
**BUGS:**

>Please describe the issue that you solved if its not filed.

>Otherwise please mention issue #id and use comma if your PR
>solves multiple issues.

**For things other than bugs:**

> Describe that thing in very short line, word limit is 200.
> Otherwise use **issue #id** if the issue was filed as **feature** request.

### CODE REVIEW

**Comment by shubhwebkul** [comment]
Hi Ashish,

Please resolve conflicts

**Comment by shubhwebkul** [comment]
Please mention what is the motive of this PR .

**Comment by Ashish503-web** [comment]
I have resolved the conflicts , I have added  the re-order functionality so that user can reorder the same product with same quantity.

**Comment by Ashish503-web** [comment]
Hi sir ,
I have solved the conflicts and   there might be an error with the composer
install in the github , I have not much experience in the github as I have
used the gitlab  so please check and let me know.
Thanks

**Comment by shubhwebkul** [comment]
No issue Ashish, you could try from cloning the project again.

File: tests/unit/Tax/Helpers/TaxCest.php [hunk 1]
```diff
@@ -63,36 +59,30 @@ public function _before(UnitTester $I)
...
-        );
 
         $this->scenario = [
-            'cart'             => Cart::getCart(),
+            'object'           => Cart::getCart(),
             'expectedTaxRates' => [
-                (string)round((float)$tax1->tax_rate, 4) => $expectedTaxAmount1,
-                (string)round((float)$tax2->tax_rate, 4) => $expectedTaxAmount2,
+                (string)round((float)$tax1->tax_rate, 4)
+                => round(11 * $product1->price * $tax1->tax_rate / 100, 2),
```
Comment (SteffenMahler) [inline tests/unit/Tax/Helpers/TaxCest.php:76]: @Ashish503-web 
Let's suppose someone wants to change the price of product 1 and 2 in this test.

What will happen if your refactoring of this test will be merged?

File: .vs/VSWorkspaceState.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (SteffenMahler) [inline .vs/VSWorkspaceState.json:1]: It seems that this folder/ files must be placed in .gitignore

**Pull request closed without merging**